# Title

MIT 7.016 Introductory Biology, Fall 2018

Course 1

* human genome only 1.5% coding for proteins
* The DNA build blocks are the same
* A, T, C, G
* 1.8 meters DNA in one cell
* add color in genetic engineering
* observe different set of protein through color changes

---

Read: Text 2.2 and 3.1, recitation section

Textbook 2.2

* covalent bonds -> sharing electrons -> strongest -> e.g. $H_2$
  * The atoms can rotate -> make up living tissues
  * single bond, $H-H$
  * double bond, $C=C$
  * triple bond, $N≡N$
  * electronegativity, the attractive force that an atomic nucleus exerts on electrons
    * The closer the electrons, the greater
    * The more positive charges, the greater
    * Two atoms with close electronegativity -> nonpolar covalent bond
    * Two atoms with different electronegativity -> polar covalent bond
* ionic attractions -> achieve stability through gain or lose electrons
  * Significantly different electronegativity
  * Na + Cl -> $NA^+ + Cl^-$
  * Positively charged ions are cations
  * Negatively charged ions are anions
  * Groups of covalently bounded atoms with electric charge are complex ions; e.g. $NH_4^+$
  * $NaCl$ -> into water -> $Na^+$ attracts the $\delta^-$ pole of water and $Cl^+-$ attracts the $\delta^+$ -> water polarity, a good biological solvent
* hydrogen bonds -> weaker interaction
  * Resulted from $\delta^-$ and $\delta^+$
  * Hydrogen bonding between water -> important fro living system
  * Hydrogen bonds play important roles in determining and maintaining the three-dimensional shapes of giant molecules such as DNA and proteins
  * hydrophobic -> water-hating -> nonpolar molecules are more attracted to one another than to water
  * hydrophilic -> water-loving -> molecules with polar covalent bounds are attracted to polar water
* van der Waals forces
  * When two molecules are in close proximity

Text 3.1

* proteins
  * polymers, constructed by covalent bonding of monomers, smaller molecules
  * formed from different combinations of 20 amino acids, all of which share chemical similarities
* carbohydrates
  * polymers, constructed by covalent bonding of monomers, smaller molecules
  * can form giant molecules by linking together chemically similar sugar monomers to form polysaccharides
* nucleic acids
  * polymers, constructed by covalent bonding of monomers, smaller molecules
  * formed from four kinds of nucleotide monomers linked together in long chains
* lipids
  * noncovalent forces maintain the interactions between the lipid monomers
  * Also form large structures from a limited set of smaller molecules
* Polyer with molecular weights > 1000 -> macromolecules -> proteins, carbohydrates, nucleic acids
* Optical isomers
  * When carbon atom has four different atoms or groups of atoms attached to it
* The function of macromolecules are directly related to their three-dimensional shapes
  * Only nucleic acids specialize in information storage and transmission
  * most macromolecules are formed by condensation and broken down by hydrolysis

---

Course 2

* main brain bilayer
* H, C, N, O, P, S -> love to make covalent bonds -> cellular mass
* Phosphorus has 5 bond
* Functional group
  * hydroxy
    ```smiles
    OCC
    ```
  * carboxyl -> lose one $H^+$
    ```smiles
    CC(=O)O
    ```
  * amine -> get one $H^+$
    ```smiles
    NCC
    ```
  * phosphates
    ```smiles
    CCOP(=O)(O)O
    ```
  * thiol
    ```smiles
    CCS
    ```
  * amine + carboxyl -> amide (remove one $H_2O$)
    ```smiles
    CC(=O)N
    ```
  * hydroxy + carboxyl -> ester (remove one $H_2O$)
    ```smiles
    CC(=O)OCC
    ```
  * phosphates + hydroxy -> phosphate ester
    ```smiles
    CCOP(=O)(OCC)O
    ```
* Non-covalent bonding -> dynamics
  * To break a covalent, 90-100 Kcal/mol
  * To break non covalent, 1-10 Kcal/mol
  * Ionic bond (salt bridge)
  * [Hydrogen bond](https://www.khanacademy.org/science/hs-chemistry/x2613d8165d88df5e:states-of-matter/x2613d8165d88df5e:intermolecular-forces/v/hydrogen-bonding) -> pole -> protein
  * hydrophobic, exclusion of water
  * van der Waals, weakly polarized bonds

---

Read Text 3.2

* Amino acids are the building blocks of proteins (Amino group and Carboxyl group)
  ```smiles
  [NH3+]C[C-]([O-])=O
  ```
  * $-COOH -> -COO^- + H^+$
  * $-NH_2 + H ^+ -> -NH_3^+$
  * amino acids are both acids and bases
  * R groups determines the structure of it, so it determines the function of protein
  * There are 20 different R groups and the charged and polar ones are hydrophilic; the nonpolar ones are hydrophobic
  * disulfide bridge: two cysteine with oxidation
  * Peptide linkage: condensation by removing one $H_2O$ and connect Amino group and Carboxyl group
  * Hydrogen bonding determines the structure of polypeptide chains ($\delta^+$ of $N-H$ and $\delta^-$ of $C=O$)
* The tertiary structure of a protein is formed by bending and folding
* Heat protein -> Denatured -> break secondary and tertiary structure -> cool down -> in some cases, the protein returns to normal structure, meaning the shape is contained in its primary structure
* The quaternary structure of a protein consists of subunits, one tertiary structure
* The shape and surface further determines the protein function
  * Only certain shape of protein can fit into another shape of protein
* Molecular Chaperones protect proteins from inappropriate binding

---

Course 3

* Semipermeable membrane
  * wrapping up by lipids
  * $O_2$ and $H_2O$ can in and out freely
  * self-healing: seal from a hole
  * Electrostatic and hydrogen-bonding on the interface
* Protein formed by amino acids (40 ~ 2000)
* Always write from N to C (amino to carboxyl)
* motif: in secondary structure, the turns of the primary structure
* Proteins are condensation polymers of $\alpha$-amino acids
* Each protein is defined by a specific sequence of monomer amino acids linked by covalent bonding
* Most native proteins fold into stable secondary and tertiary structures
* Folding is defined by thermodynamic stability and the sum of many non-covalent interactions
* Protein subunit may also associate through non-covalent interactions into quaternary structures

---

Read text: 8.1, 8.3 - 8.5 (read the summary first, the summary is in the end of the chapter)

https://www.youtube.com/watch?v=yk14dOOvwMk

textbook 8.1

* The usable energy is free energy; The unusable energy is entropy
  * $H = G + TS$ (total energy = usable energy + unusable energy, T is absolute temperature and S is entropy)
  * $\triangle G_{reaction} = G_{product} - G_{reactants}$ => $\triangle G = \triangle H - T\triangle S$
* The potential energy is the energy of state; Kinetic energy is the energy of motion, which can do work
  * Potential: chemical bonds, concentration gradient, electric charge imbalance
  * Kinetic: heat
  * Potential <-> Kinetic
* Laws of thermodynamics
  * Energy cannot be created or destroyed
  * Energy transformations decrease the amount of energy for work and increase disorder
* The change in free energy determines its point of chemical equ. (The forward and reverse reactions proceed at the same rate)
* An exergonic reaction -> release free energy, -$\triangle G$; An endergonic reaction -> consumes free energy, +$\triangle G$
* Metabolism = the sum of all biochemical reactions; Catabolic reactions, breakdown of complex molecules and release energy; Anabolic reactions build complexity in the cell and are endergonic.
* Catabolic: Complex molecules -> free energy + small molecules
* Anabolic: Free energy + small molecules -> complex molecules
* Standard laboratory condition: $25^oC$, one atmosphere pressure, one molar concentration of the solutes, pH7. The standard free energy change under standard condition is $\triangle G^{o'}$

textbook 8.3

* The more negative the $\triangle G$ is, the further the reaction proceeds toward completion.
* Catalyst does not cause a reaction to occur, but merely increases the rate of the reaction
* To speed up a reaction, an energy barrier must be overcome
* Enzymes lower the energy barrier but do not affect equilibrium.

textbook 8.4

* Most enzymes are much larger than their substrates
* The active site is specific to the substrates
  * a substrate can be oriented correctly, chemically modified, or strained
* Induced fit: The change in enzyme shape caused by substrate binding
  * Induced fit explains why enzymes are so large
* Some enzymes require other cofactors to carry out catalysis
  * A coenzyme can be considered a substrate, it is changed by the reaction and then released from the enzyme
* Substrate concentration affects the rate of enzyme-catalyzed reaction

textbook 8.5

* Metabolism is organized into pathways in which the product of one reaction is a reactant for the next reaction.
* ENzymes can be regulated by inhibitors
  * If an inhibitor covalently binds to certain side chains at the active site of an enzyme, it will per- manently inactivate the enzyme by destroying its capacity to interact with its normal substrate.
  * The inhibition is reversible because if the con- centration of substrate is increased or if the concentration of inhibitor is reduced, the substrate is more likely to bind, and the enzyme is active again.
  * A competitive inhibitor binds to the active site, preventing substrate binding.
  * An uncompetitive inhibitor binds to the enzyme–substrate complex, preventing release of products.
  * A noncompetitive inhibitor binds at a site other than the active site, changing enzyme structure so that normal substrate binding cannot occur.
* The first reaction in a metabolic pathway is referred to as the commitment step.
* Enzymes are highly sensitive to changes in temperature and pH.
  * pH decreases -> more H+ -> react with $COO^-$ -> no longer negatively charged -> no longer interact with positively charged groups in the protein -> the folding of the protein may be altered
  * Temperature increase -> the reaction rate increase -> temperature too high -> reaction drops abruptly
  * Most enzymes in humans are more stable at high temperatures than those of the bacteria that infect us, so that a moderate fever tends to denature bacterial enzymes, but not our own.

